scipy/optimize/_trustregion_exact.py

```diff
@@ -0,0 +1,413 @@
+"""Nearly exact trust-region optimization subproblem."""
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.linalg import (norm, get_lapack_funcs, solve_triangular,
+                          cho_solve)
+from ._trustregion import (_minimize_trust_region, BaseQuadraticSubproblem)
+
+__all__ = ['estimate_smallest_singular_value',
```

Comment:
The fact that `__all__` doesn't contain `_minimize_trustregion_exact` looks not logical.